scripts/deploy_assisted_service.sh

```diff
@@ -117,6 +115,9 @@ EOF
 
     ./assisted-service/deploy/operator/deploy.sh
     echo "Installation of Assisted Install operator passed successfully!"
+    PATCH_ROUTE=$(kubectl get routes -n assisted-installer --no-headers | awk '{print $2}')
+    sed -i "s/${ROUTE}/${PATCH_ROUTE}/g" "${HOME}/.test-infra/etc/nginx/conf.d/http_localhost.conf"
+    sleep 60
```

Comment:
Figure out how to do it without sleep (instant NGINX reload?)

Reply:
+1

Reply:
added a comment for now